src/csharp/Microsoft.Spark/ML/Feature/CountVectorizerModel.cs

```diff
@@ -0,0 +1,170 @@
+        public double GetMaxDF() => (double)_jvmObject.Invoke("getMaxDF");
+        
+        /// <summary>
+        /// Gets the minimum number of different documents a term must appear in to be included in
+        /// the vocabulary. If this is an integer greater than or equal to 1, this specifies the
+        /// number of documents the term must appear in; if this is a double in [0,1), then this
+        /// specifies the fraction of documents.
+        /// </summary>
+        /// <returns>The minimum document term frequency</returns>
+        public double GetMinDF() => (double)_jvmObject.Invoke("getMinDF");
+
+        /// <summary>
+        /// Gets the filter to ignore rare words in a document. For each document, terms with
+        /// frequency/count less than the given threshold are ignored. If this is an integer
+        /// greater than or equal to 1, then this specifies a count (of times the term must appear
+        /// in the document); if this is a double in [0,1), then this specifies a fraction (out of
+        /// the document's token count).
+        ///
+        /// Note that the parameter is only used in transform of CountVectorizerModel and does not
+        /// affect fitting.
+        /// </summary>
+        /// <returns>Minimum term frequency of type double.</returns>
+        public double GetMinTF() => (double)_jvmObject.Invoke("getMinTF");
+
+        /// <summary>
+        /// Sets the filter to ignore rare words in a document. For each document, terms with
+        /// frequency/count less than the given threshold are ignored. If this is an integer
+        /// greater than or equal to 1, then this specifies a count (of times the term must appear
+        /// in the document); if this is a double in [0,1), then this specifies a fraction (out of
+        /// the document's token count).
+        ///
+        /// Note that the parameter is only used in transform of CountVectorizerModel and does not
+        /// affect fitting.
+        /// </summary>
+        /// <param name="value">Minimum term frequency of type double.</param>
+        /// <returns>
+        /// New <see cref="CountVectorizerModel"/> with the min term frequency set
+        /// </returns>
+        public CountVectorizerModel SetMinTF(double value) =>
+            WrapAsCountVectorizerModel((JvmObjectReference)_jvmObject.Invoke("setMinTF", value));
+        
+        /// <summary>
+        /// Gets the max size of the vocabulary. <see cref="CountVectorizerModel"/> will build a
+        /// vocabulary that only considers the top vocabSize terms ordered by term frequency across
+        /// the corpus.
+        /// </summary>
+        /// <returns>The max size of the vocabulary of type int.</returns>
+        public int GetVocabSize() => (int)_jvmObject.Invoke("getVocabSize");
+        
+        private static CountVectorizerModel WrapAsCountVectorizerModel(object obj) => 
```

Comment:
If you are casting `obj` to `JvmObjectReference` in this function, you don't need to cast in the caller.